fetch: fix typo in the README (#97)
cockroachdb/molt: README.md

```

### Edge case
#### MacOS + CockroachDB as source within Docker container

If you encounter an error similar to the following, please contact the support team.

```
ERROR: AS OF SYSTEM TIME: cannot specify timestamp in the future (1701836988.000000000,0 > 1701836987.322737000,0) (SQLSTATE XXUUU)
```

> This error is due to the fact that with MacOS as the runtime OS, Docker may have indeterministic time drift from the host machine.[[1](https://github.com/cockroachdb/molt/issues/93)]Because we run a `SELECT ... AS OF SYSTEM TIME` query to iterate content from the target table, time drift can cause a `cannot specify timestamp in the future` error when using `molt fetch` to export data from a CockroachDB cluster within a container.

Reviewer: ```suggestion
This error is due to the fact that with MacOS as the runtime OS, Docker may have indeterministic time drift from the host machine.[[1](https://github.com/cockroachdb/molt/issues/93)] Because we run a `SELECT ... AS OF SYSTEM TIME` query to iterate content from the target table, time drift can cause a `cannot specify timestamp in the future` error when using `molt fetch` to export data from a CockroachDB cluster within a container.
```

Author: Done!

done
```

> Such a time drift can leads to the `cannot specify timestamp in the future` error when using `molt fetch` to export data from a CockroachDB cluster within a container, as we run a `SELECT ... AS OF SYSTEM TIME` query to iterate content from the target table.

Reviewer: I also suggest moving this sentence directly after "This error is due to the fact that with MacOS as the runtime OS, Docker may have indeterministic time drift from the host machine."

```suggestion
Because we run a `SELECT ... AS OF SYSTEM TIME` query to iterate content from the target table, time drift can cause a `cannot specify timestamp in the future` error when using `molt fetch` to export data from a CockroachDB cluster within a container.
```
